internal/nginx/extensions_test.go

```diff
@@ -10,6 +10,9 @@ func TestParseLBMethod(t *testing.T) {
 		{"least_conn", "least_conn"},
 		{"round_robin", ""},
 		{"ip_hash", "ip_hash"},
+		{"random", "random"},
+		{"random two", "random two"},
+		{"random two least_conn", "random two least_conn"},
 		{"hash $request_id", "hash $request_id"},
```

Comment:
please also test for some of invalid input:
`random one`, `random two ip_hash`, `random two least_time`